dask/dataframe/io/io.py

```diff
@@ -101,7 +101,7 @@ def from_array(x, chunksize=50000, columns=None):
     return new_dd_object(dsk, name, meta, divisions)
 
 
-def from_pandas(data, npartitions=None, chunksize=None, sort=True, name=None):
+def from_pandas(data, npartitions=10, chunksize=None, sort=True, name=None):
```

Comment:
I'm not sure why the default value of `npartitions` was changed here. Could you elaborate on your reasoning for this @HSR05?